backend/src/main/twirl/MainFutureAkka.scala.txt

```diff
@@ -13,7 +16,7 @@ object Main {
   def main(args: Array[String]): Unit = {
     implicit val actorSystem: ActorSystem = ActorSystem()
 
-    val route = AkkaHttpServerInterpreter().toRoute(List(ApiDefinitions.helloServerEndpoint))
+    val route = AkkaHttpServerInterpreter().toRoute(ApiDefinitions.helloServerEndpoint@if(documentationAdded) { :: ApiDefinitions.docEndpoints})
```

Comment:
Maybe it would be cleaner to rename `ApiDefinitions` -> `Endpoints`, and there have:
```
object Endpoints {
  val hello = ...
  val docs = ...
  val all = List(hello) ++ docs
}
```

and then reference `Endpoints.all` here ?